### PR TITLE
add nginx to Recommendations

### PR DESCRIPTION
**Pull request by mhf-ir**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

File: docs/Recommendations.md [hunk 1]
```diff
@@ -161,6 +163,83 @@ backend static-backend
...
 [haproxy]: https://www.haproxy.org/
 
+### Nginx
+
+```nginx
+upstream fastify_app {
+  # more info: http://nginx.org/en/docs/http/ngx_http_upstream_module.html
+  server 10.10.11.1:80;
+  server 10.10.11.2:80;
+  server 10.10.11.3:80 backup;
```
Comment (mcollina) [inline docs/Recommendations.md:173]: I think we should add some docs about folks configuring this ip address.

Comment (mhf-ir) [reply docs/Recommendations.md:173]: I just take it from haproxy example.

It's better to be simpler example focus on TLS offloader and load balancing technique. HAProxy example is more complicated i think.

**Review by jsumners — COMMENTED** [review]
This really needs a lot more documentation. Please re-review the HAProxy example. It uses inline comments to explain _everything_ about the configuration. This NGinx example assumes the reader knows a lot about NGinx.

**Pull request merged**